@kiva/kv-components/vue/KvCheckbox.vue

```diff
@@ -105,73 +106,75 @@ export default {
 			default: true,
 		},
 	},
-	data() {
-		return {
-			uuid: `kvc-${nanoid(10)}`,
-			isChecked: false,
-		};
-	},
-	computed: {
-		inputListeners() {
-			return {
-				// Pass through any listeners from the parent to the input element, like blur, focus, etc.
-				// https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components
-				...this.$listeners,
-				// ...except for the listener to the 'change' event which is emitted by this component
-				change: () => {},
-			};
-		},
-	},
-	watch: {
-		checked() {
-			this.setChecked();
-		},
-	},
-	mounted() {
-		this.uuid = `kvc-${nanoid(10)}`;
-	},
-	created() {
-		this.setChecked();
-	},
-	methods: {
-		onChange(event) {
-			// get the input[type=checkbox] state
-			const isChecked = event.target.checked;
+	emits: [
+		'change',
+		'update:modelValue',
+	],
+	setup(props, { emit }) {
+		const {
+			checked,
+			value,
+		} = toRefs(props);
```

Comment:
Because `checked` and `value` are refs, `.value` needs to be used to get their current value. For example:
```
checkboxValue = [...checked.value, event.target.value];
```

Reply:
Fixed, thanks!